IdentityServer4.Contrib.RedisStore/IdentityServer4.Contrib.RedisStore.csproj

```diff
@@ -30,9 +30,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="IdentityServer4" Version="3.0.*" />
-    <PackageReference Include="IdentityServer4.Storage" Version="3.0.*" />
-    <PackageReference Include="StackExchange.Redis" Version="2.0.*" />
+    <PackageReference Include="IdentityServer4" Version="3.1.1" />
+    <PackageReference Include="IdentityServer4.Storage" Version="3.1.1" />
+    <PackageReference Include="StackExchange.Redis" Version="2.0.519" />
```

Comment:
I may agree over this hardcoding only, the other libraries should still be supporting older version, no point of upgrading them.

Reply:
Yes you are right.